packages/stories/src/index.tsx

```diff
@@ -84,9 +110,11 @@ export default function Stories({
   }
 
   useEffect(() => {
-    if (selectedStory) {
-      onStoryChange(selectedStory.index);
-    }
+    if (!selectedStory) return;
+
+    onStoryChange(selectedStory.index);
+
+    preloadStory(selectedStory.index + 1);
```

Comment:
i think we dont need it here. the above hook is preloading the story on every currentIndex Change.

Reply:
Is not entering to that hook :S